charts/kured/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.6.1"
 description: A Helm chart for kured
 name: kured
-version: 2.4.2
```

Comment:
Sure, I can push a separate PR for the Helm chart change. Would the appVersion change to 1.7.0 in this case since the change adds a new parameter to the application?

Reply:
Yeah, it's a new feature. We are not stricto sensu respecting semver, but we are very close:
We try to bundle the new features in a new minor bump (x.Y.Z).
Those minor bumps are used to indicate the supported kubernetes version change, so overall new features are provided with the next kubernetes support bump.

If you're interested to merge this soon, we should work on it now, as kubernetes 1.21 is out, and we'll probably release not long after kind for 1.21 is ready :)

Reply:
Thanks. I'll move the chart update to a separate PR.